src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1453,5 +1453,37 @@ class C
                 state.AssertNoCompletionSession()
             End Using
         End Sub
+
+        <WorkItem(1594, "https://github.com/dotnet/roslyn/issues/1594")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub NoPreselectionOnSpaceWhenAbuttingWord()
```

Comment:
Should the test be called NoSessionOnSpace... since that's what it's actually asserting?